keyboards/meletrix/zoom65/keymaps/65_ansi/rules.mk

```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
```

Comment:
VIA should only be enabled for the VIA layout.  

Additionally, any sort of options for layout should be determined by the via json file, so it can be configured in via properly.

```suggestion
```

Reply:
Deleted